Sources/Markdown/Block Nodes/Block Container Blocks/OrderedList.swift

```diff
@@ -29,7 +29,26 @@ public extension OrderedList {
     // MARK: ListItemContainer
 
     init<Items: Sequence>(_ items: Items) where Items.Element == ListItem {
-        try! self.init(.orderedList(parsedRange: nil, items.map { $0.raw.markup }))
+        try! self.init(.orderedList(parsedRange: nil, items.map { $0.raw.markup }, start: nil))
+    }
+
+    /// The starting index for the list.
+    ///
+    /// If this is `nil`, the list will start at the default value of 1.
```

Comment:
I think the phrasing could be improved here, I think what we mean is that if `start` is `nil`, it's up to the client to pick a start index? (1, most likely)